ignite/cmd/network_chain_join.go

```diff
@@ -35,6 +38,7 @@ func NewNetworkChainJoin() *cobra.Command {
 
 	c.Flags().String(flagGentx, "", "Path to a gentx json file")
 	c.Flags().String(flagAmount, "", "Amount of coins for account request")
```

Comment:
```suggestion
	c.Flags().String(flagAmount, "", "Amount of coins for account request (ignored if coordinator has fixed the account balances or if --no-acount flag is set)")
```

This extra information could also be moved in the command `Long` field, like for instance in the cosmos bank command : 

https://github.com/cosmos/cosmos-sdk/blob/5013933e6f960274804d7c71492f4b7da04a25d9/x/bank/client/cli/tx.go#L40-L42